orchestrator/dags/utils.py

```diff
@@ -1,12 +1,38 @@
 import settings
+import requests
 
 
-def generate_api_warszawa_url(api_key: str, resource_id: str, **kwargs):
-    """
-    Returns url to Warsaw Public Transport API with given parameters as query strings
-    """
-    url = settings.API_WARSZAWA_URL
-    url += f'?apikey={api_key}&resource_id={resource_id}'
-    for k, v in kwargs.items():
-        url += f'&{k}={v}'
-    return url
+class ApiWarsawUrl:
+    BASE_URL = 'https://api.um.warszawa.pl/'
+    ENDPOINT_DBSTORE = 'api/action/dbstore_get/'
+    ENDPOINT_BUSESTRAMS = 'api/action/busestrams_get/'
+
+    def __init__(self, apikey):
+        self._client = requests.session()
+        self._apikey = apikey
+
+    def _get(self, url, params=None):
+        if params is None:
+            params = {}
+        default_params = {"apikey": self._apikey}
+        default_params.update(params)
+        response = self._client.get(
+            f"{ApiWarsawUrl.BASE_URL}{url}", params=default_params
+        )
+        response.raise_for_status()
+        return response.status_code, response.json()
+
+    def get_dbstore(self, id=None):
+        params = {}
+        if id is not None:
+            params["id"] = id
+        self._dbstore = self._get(ApiWarsawUrl.ENDPOINT_DBSTORE, params)
+        return self._dbstore
+
+    def get_busestrams(self, resource_id=None, type=None):
+        params = {}
+        if resource_id is not None and type is not None:
+            params["resource_id"] = resource_id
+            params["type"] = type
+        self._busestrams = self._get(ApiWarsawUrl.ENDPOINT_BUSESTRAMS, params)
+        return self._busestrams
```

Comment:
```suggestion
        return self._get(ApiWarsawUrl.ENDPOINT_BUSESTRAMS, params)
        
```
The `_busestrams` is not used across the class, so we don't need to define it.